resources/assets/js/bbcode-set.ts

```diff
@@ -72,7 +76,17 @@ export const bbCodeSet = {
 				return markitup.selection;
 			},
 		},
-		{className: 'ico picture', name: 'Afbeelding', replaceWith: '[img][![Url]!][/img]'},
+		{
+			className: 'ico picture', name: 'Plaatje',
+			closeWith: (markitup: any) => {
+				axios.get('/plaatjes/upload').then((response) => {
+					domUpdate(response.data);
```

Comment:
Misschien hier de response checken en domupdate aanroepen als er een modal is en anders de insertPlaatje actie doen.

Reply:
Maar je krijgt hier altijd een modal terug toch? De insertplaatje gebeurt bij het submitten van de modal.

Reply:
Misschien de onsubmit event op de modal overriden en dan het resultaat gebruiken om een plaatje te inserten.

Reply:
Ohja tuurlijk.